Ginger/GingerCore/ALM/Qtest/ExportToQtest.cs

```diff
@@ -0,0 +1,594 @@
+                                            }
+                                            System.IO.Directory.Delete(activGroup.TempReportFolder, true);
+                                            //Creating the Zip file - finish
+                                            //Attaching Zip file - start
+                                            AttachmentFactory attachmentFactory = (AttachmentFactory)run.Attachments;
+                                            TDAPIOLELib.Attachment attachment = (TDAPIOLELib.Attachment)attachmentFactory.AddItem(System.DBNull.Value);
+                                            attachment.Description = "TC Ginger Execution HTML Report";
+                                            attachment.Type = 1;
+                                            attachment.FileName = zipFileName;
+                                            attachment.Post();
+
+                                            //Attaching Zip file - finish
+                                            System.IO.File.Delete(zipFileName);
+                                        }
+                                    }
+
+
+                                    //create run with activities as steps
+                                    run.CopyDesignSteps();
+                                    run.Post();
+                                    StepFactory stepF = run.StepFactory;
+                                    List stepsList = stepF.NewList("");
+
+                                    //int i = 0;
+                                    int index = 1;
+                                    foreach (Step step in stepsList)
+                                    {
+                                        //search for matching activity based on ID and not order, un matching steps need to be left as No Run
+                                        int stepDesignID = (stepsList[index]).Field("ST_DESSTEP_ID");
+                                        Activity matchingActivity = activities.Where(x => x.ExternalID == stepDesignID.ToString()).FirstOrDefault();
+                                        if (matchingActivity != null)
+                                        {
+                                            switch(matchingActivity.Status)
+                                            {
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed:
+                                                    step.Status = "Failed";
+                                                    List<IAct> failedActs= matchingActivity.Acts.Where(x => x.Status == Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed).ToList();
+                                                    string errors = string.Empty;
+                                                    foreach (Act act in failedActs) errors += act.Error + Environment.NewLine;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)